system_tests/test_service_account.py

```diff
@@ -47,19 +45,3 @@ def test_refresh_success(http_request, credentials, token_info):
             "https://www.googleapis.com/auth/userinfo.profile",
         ]
     )
-
-def test_iam_signer(http_request, credentials):
```

Comment:
Why is this test removed? Please add it back in.

Reply:
These tests are no longer contained in master, I don't recall removing them from the test file.